.github/workflows/package.yml

```diff
@@ -0,0 +1,300 @@
+name: Package and release
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
The workflow needs a tag which will be the release to know where to push the artifacts

Reply:
We anyway need the option to choose a tag for a release, no? The current [version numbers](https://github.com/clementperon/PcapPlusPlus/releases) are just for tests... 
Maybe this can help? 
https://github.com/community/community/discussions/45871

Reply:
When you push the tag this workflow is then triggered, create an associated release to the tag  and push the artifacts.

The release will have the tag name that you can edit after

Reply:
thanks for clarifying, I think I got it now